cypress/fixtures/leave.json

```diff
@@ -0,0 +1,9 @@
+{
+    "comment": null,
+    "duration": {
+        "type": "full_day"
+    },
+    "fromDate": "2023-10-31",
+    "leaveTypeId": 8,
+    "toDate": "2023-10-31"
```

Comment:
For the dates there is a library called moment -  https://momentjs.com/  - we used it to declare the dates,
for these dates for the future case, when you write the test in that day that was worked but when run test after (31-10-2023) Not worked cuz 31 was past dates so not everyday the test will working , so please use moment